Covinator/Covinator/Pages/modernaVaccination.cshtml

```diff
@@ -1,34 +1,31 @@
 ﻿@page
 @model Covinator.Pages.modernaVaccinationModel
 @{
+    ViewData["Title"] = "Moderna Vaccine";
     var modernaVaccineDistributionAllocationss = (QuickType.ModernaVaccineDistributionAllocations[])ViewData["ModernaVaccineDistributionAllocations"];
 }
 
 <div class="text-center">
-    <h1 class="display-4">Moderna Vaccine Distribution Allocations</h1>
-    <hr />
-    <table border="1" weidth="90%">
```

Comment:
Removed inline styling and added same class as used in other tables for a uniform UI

Reply:
Good.